src/components/daydream-controls.js

```diff
@@ -18,14 +16,16 @@ var GAMEPAD_ID_PREFIX = 'Daydream Controller';
  */
 module.exports.Component = registerComponent('daydream-controls', {
   schema: {
-    hand: {default: DEFAULT_HANDEDNESS}, // This informs the degenerate arm model.
+    hand: {default: ''}, // This informs the degenerate arm model.
```

Comment:
I've been thinking about `daydream` a bit more. Until daydream supports multiple controllers we should make this component a `singleton` somehow. I propose that in the `play` method we do:
var daydreamEl = this.sceneEl.querySelector(['daydream-controls']);
````javascript
if (daydreamEl && daydreamEl.components['daydream-controls']) { 
  console.warn('Detected multiple instances of daydream-controls component. Only one controller is currently supported');
  return;
}
````